apps/keyboard/README.md

```diff
@@ -0,0 +1,15 @@
+# Keyboard
+
+Keyboard app work closely with system to handle Keyboard layout and multiple input methods(IME).
```

Comment:
Suggestiton:
Keyboard app enables the user to input with different keyboard layouts and input methods (IME).
The input management (in system app) will launch keyboard app within input window and manage its life cycle.